tracer/src/Datadog.Trace/Configuration/ConfigurationKeys.Debugger.cs

```diff
@@ -106,6 +106,12 @@ internal static class Debugger
             /// <seealso cref="DebuggerSettings.RedactedIdentifiers"/>
             public const string RedactedIdentifiers = "DD_DYNAMIC_INSTRUMENTATION_REDACTED_IDENTIFIERS";
 
+            /// <summary>
+            /// Configuration key for set of identifiers that are excluded from redaction decisions.
+            /// </summary>
+            /// <seealso cref="DebuggerSettings.RedactedExcludedIdentifiers"/>
+            public const string RedactedExcludedIdentifiers = "DD_DYNAMIC_INSTRUMENTATION_REDACTION_EXCLUDED_IDENTIFIERS";
```

Comment:
If this is related to the setting above, should the names match to avoid user error? `REDACTED` above vs `REDACTION` here.

```
DD_DYNAMIC_INSTRUMENTATION_REDACTED_IDENTIFIERS
                                 ^
                                 |
                                 v
DD_DYNAMIC_INSTRUMENTATION_REDACTION_EXCLUDED_IDENTIFIERS
```

Reply:
It makes sense. I must stick to the names in the RFC but I have asked that in the team chanel, I'm waiting to conclusion.

Reply:
We decided to support both